instrumentation/common/src/main/java/com/splunk/opentelemetry/servertiming/ServerTimingHeader.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.opentelemetry.servertiming;
+
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.propagation.W3CTraceContextPropagator;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.propagation.TextMapPropagator.Setter;
+import io.opentelemetry.instrumentation.api.config.Config;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Adds {@code Server-Timing} header (and {@code Access-Control-Expose-Headers}) to the HTTP
+ * response. The {@code Server-Timing} header contains the traceId and spanId of the server span.
+ */
+public final class ServerTimingHeader {
+  public static final String SERVER_TIMING = "Server-Timing";
+  public static final String EXPOSE_HEADERS = "Access-Control-Expose-Headers";
+
+  public static boolean shouldEmitServerTimingHeader() {
+    return Config.get().getBooleanProperty("splunk.context.server-timing.enabled", false);
+  }
+
+  public static <RS> void setHeaders(Context context, RS response, Setter<RS> headerSetter) {
+    if (!Span.fromContext(context).getSpanContext().isValid()) {
+      return;
+    }
+
+    headerSetter.set(response, SERVER_TIMING, toHeaderValue(context));
+    headerSetter.set(response, EXPOSE_HEADERS, SERVER_TIMING);
+  }
+
+  private static String toHeaderValue(Context context) {
+    Map<String, String> traceContextHeaders = new HashMap<>();
+    W3CTraceContextPropagator.getInstance().inject(context, traceContextHeaders, Map::put);
+    return String.format("traceparent;desc=\"%s\"", traceContextHeaders.get("traceparent"));
```

Comment:
I think I've recently heard people say that String.format is significantly slower than just plain old string concatenation these days. I don't have data...just memories of people making this comment.

Reply:
I've heard that too, actually -- but `String.format()` has been my default way to do this for a long time, so I forget myself and keep using it...
Anyway, I'll change that to string concatenation. Thanks!